cmds/status/status.go

```diff
@@ -0,0 +1,267 @@
+package status
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"os/user"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/fatih/color"
+	"github.com/taskcluster/taskcluster-cli/root"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	manifestURL = "https://references.taskcluster.net/manifest.json"
+)
+
+var (
+	pingURLs  PingURLs
+	validArgs []string
+	cacheFile = CacheFilePath()
+)
+
+type (
+	PingURLs map[string]string
+
+	CachedURLs struct {
+		LastUpdated time.Time `json:"lastUpdated"`
+		PingURLs    PingURLs  `json:"pingURLs"`
+	}
+
+	PingResponse struct {
+		Alive  bool    `json:"alive"`
+		Uptime float64 `json:"uptime"`
+	}
+
+	API struct {
+		BaseURL string     `json:"baseUrl"`
+		Entries []APIEntry `json:"entries"`
+	}
+
+	APIEntry struct {
+		Name  string `json:"name"`
+		Route string `json:"route"`
+	}
+)
+
+// CacheFilePath returns the file system path to the cache file storing the ping URLs
+func CacheFilePath() string {
+	// 1. find out where home directory is (panic in case of error)
+	usr, err := user.Current()
+	if err != nil {
+		panic(err)
+	}
+	home := usr.HomeDir
+
+	// 2. return file path <home directory>/ .taskcluster-cli/ cmds/ status / cache.json
+	return filepath.Join(home, ".taskcluster-cli", "cmds", "status", "cache.json")
```

Comment:
@djmitche Technically, this is not config (it should not be editable by a human). It is variable data, so doesn't belong in a config directory - but possibly could go in e.g. `/var`.